### ISSUE TITLE
After update my root key is not accepted by the server. 

### ISSUE THREAD

**Issue by yazzou** [omega8cc/boa#1630]
I have launched the update script as usual but now i am unable to connect as root on my server as my key is no more accepted. 
It does not recognize the certificate. 
Apparently, during the update, the key (from the host side) has been updated.

**Comment by dserrato** [comment]
Are you receiving the possible man in the middle warning? If so, I've used that key that the warning provides and added it to my known_hosts file wit the server ip. 
example:
123.222.333.123 SSD231ALKAFJLDJFABLAHBLAHBLAH

Also, I removed the previous reference to the server key. 

Then, when logging back into the server, it will ask to accept and it includes the server into your known_hosts file. 

I then removed the temporary key (example above) and I was able to log in normally after that. 

If this is the case for you, I hope it helps.
